packages/app-store/make/api/subscriptions/deleteSubscription.ts

```diff
@@ -0,0 +1,39 @@
+import type { NextApiRequest, NextApiResponse } from "next";
+import z from "zod";
+
+import { deleteSubscription } from "@calcom/app-store/_utils/nodeScheduler";
+import findValidApiKey from "@calcom/features/ee/api-keys/lib/findValidApiKey";
+import { defaultHandler, defaultResponder } from "@calcom/lib/server";
+
+const querySchema = z.object({
+  apiKey: z.string(),
+  id: z.string(),
+});
+
+async function handler(req: NextApiRequest, res: NextApiResponse) {
```

Comment:
when is deleteSubscribtion called in make? I don't ever see the webhook deleted in the database

Reply:
In the sidebar, there's a section called `Webhooks`. You need to select the webhook from there and delete each individually. I've added a line about this in the readme.